ProcessLib/SmallDeformation/SmallDeformationFEM.h

```diff
@@ -209,6 +210,9 @@ class SmallDeformationLocalAssembler
         auto local_b = MathLib::createZeroedVector<NodalDisplacementVectorType>(
             local_b_data, local_matrix_size);
 
+        if (_is_deactivated)
+            return;
```

Comment:
Maybe add a comment `// return after local_Jac and local_b have been resized and zeroed in order to avoid complaints from the VectorMatrixAssembler` (if that is the case)

Reply:
Changed.